tests/application/assets/StreetLamp.ts

```diff
@@ -0,0 +1,13 @@
+import { AssetModelDefinition } from "../../../index";
+
+export const streetLampAssetDefinition: AssetModelDefinition = {
+  measures: [
+    {
+      name: "position",
+      type: "position",
+    },
+  ],
+  metadataMappings: {
+    street: { type: "keyword" },
+  },
```

Comment:
As the position is a constant it's better in the metadatas (cf [iot-platform](https://github.com/kuzzleio/iot-platform/blob/3-dev/apps/api/lib/modules/public_lighting/assets/StreetLamp.ts))
```suggestion
  measures: [
    {
      name: "brightness",
      type: "brightness",
    },
    {
      name: "powerConsumption",
      type: "powerConsumption",
    },
  ],
  metadataMappings: {
    position: { type: "geo_point" },
    street: { type: "keyword" },
  },
```